### PR TITLE
0.23.0 blog post

### PR DESCRIPTION
**Pull request by patriksvensson**
Will write this ASAP.

### CODE REVIEW

**Comment by gep13** [comment]
@patriksvensson as mentioned in this PR https://github.com/cake-build/website/pull/414, I think it makes sense to reference the new documentation in the blog post, so people can get more information about it.

**Comment by gep13** [comment]
@patriksvensson if we could get this documented as well: https://github.com/cake-build/website/issues/410 as part of the release, and mentioned in the blog post, I think that would be great too 😄

**Comment by patriksvensson** [comment]
@gep13 @devlead Could you take a quick look at this? Thanks!

File: input/blog/2017-10-11-cake-v0.23.0-released.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Cake v0.23.0 released
+category: Release Notes
+author: patriksvensson
+---
+
+Version 0.23.0 have been released!
```
Comment (gep13) [inline input/blog/2017-10-11-cake-v0.23.0-released.md]: `has been released`

Comment (devlead) [reply input/blog/2017-10-11-cake-v0.23.0-released.md]: I've corrected and rebased.

File: input/blog/2017-10-15-cake-v0.23.0-released.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        Information("Copied {0} bytes into memory.",
+            outputStream.Length
+            );
+    }
+});
+```
+
+# Reverse task dependencies
+
+Ever wished you could define your task as a dependency of another task - i.e. a reversed dependency relataionship? The good news is that you now can.
```
Comment (devlead) [inline input/blog/2017-10-15-cake-v0.23.0-released.md]: Typo `relataionship` I'll fix and rebase again.

File: input/blog/2017-10-15-cake-v0.23.0-released.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- [devlead](https://github.com/devlead)
+- [dracan](https://github.com/dracan)
+- [ErikSchierboom](https://github.com/ErikSchierboom)
+- [patriksvensson](https://github.com/patriksvensson)
+- [adamhathcock](https://github.com/adamhathcock)
+- [mholo65](https://github.com/mholo65)
+- [jpreese](https://github.com/jpreese)
+- [Redth](https://github.com/Redth)
+- [joaoasrosa](https://github.com/joaoasrosa)
+- [#](Giovanni Van Geel)
```
Comment (devlead) [inline input/blog/2017-10-15-cake-v0.23.0-released.md]: This link goes nowhere I'll correct.

**Comment by devlead** [comment]
@patriksvensson  your changes have been merged, thanks for your contribution 👍

**Pull request merged**